### PR TITLE
v8 10.4.132.22

### PR DESCRIPTION
**Pull request by chiefmikey**
Created with `brew bump-formula-pr`.

`resource` blocks may require updates.

### CODE REVIEW

**Comment by carlocab** [comment]
`resource`s need updating.

**Comment by chiefmikey** [comment]
I went through the tag's DEPS file and made sure all resources are now matching.

File: Formula/v8.rb [hunk 1]
```diff
@@ -61,7 +61,7 @@ class V8 < Formula
 
   resource "v8/third_party/icu" do
     url "https://chromium.googlesource.com/chromium/deps/icu.git",
-        revision: "1fd0dbea04448c3f73fe5cb7599f9472f0f107f1"
+        revision: "165825933050109d8331d0faa56cc9f52460fbbf"
```
Comment (carlocab) [inline Formula/v8.rb:64]: This doesn't appear to be a valid ref for some reason. Weird.

Comment (chiefmikey) [reply Formula/v8.rb:64]: Ya for real, I found the [log of it being replaced](https://groups.google.com/g/v8-reviews/c/ROeN0ry6R6A) but not sure where it came from to begin with. Closing this in favor of #109593 which hopefully doesn't have any non-existent refs!

**Pull request closed without merging**